src/diffusers/models/resnet.py

```diff
@@ -25,560 +25,17 @@
 from .attention_processor import SpatialNorm
 from .lora import LoRACompatibleConv, LoRACompatibleLinear
 from .normalization import AdaGroupNorm
-
```

Comment:
💯 percent not sure why this important change is now being displayed:

https://github.com/huggingface/diffusers/blob/2cac34f41929edb4041cdea9c67f3cfa7cc5dc66/src/diffusers/models/resnet.py#L28